pages/profile/planetcash/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement, useState } from 'react';
+import TopProgressBar from '../../../src/features/common/ContentLoaders/TopProgressBar';
+import UserLayout from '../../../src/features/common/Layout/UserLayout/UserLayout';
+import Head from 'next/head';
+import PlanetCash, {
+  PlanetCashTabs,
+} from '../../../src/features/user/PlanetCash';
+import i18next from '../../../i18n';
+
+const { useTranslation } = i18next;
+
+export default function PlanetCashPage(): ReactElement {
+  const { t, ready } = useTranslation('me');
+  const [progress, setProgress] = useState(0);
+
```

Comment:
Add the cleanup function added in the `/profile/planetcash/new` and `/profile/planetcash/transactions` here as well.

```
 // Cleanup function to reset state and address Warning: Can't perform a React state update on an unmounted component.
  useEffect(() => {
    return () => {
      setProgress(0);
    };
  }, []);
```